### PR TITLE
Add Occlusion Query Scissor Test

### PR DESCRIPTION
**Pull request by greggman**
The dEQP occlusionquery_strict test failed on Chrome but
not on Safari. This test provokes the same bug in both
Chrome and Safari.

See https://anglebug.com/7157

### CODE REVIEW

File: sdk/tests/conformance2/query/occlusion-query-scissor.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<!--
+Copyright (c) 2019 The Khronos Group Inc.
```
Comment (kenrussell) [inline sdk/tests/conformance2/query/occlusion-query-scissor.html:2]: Please update to 2022.

File: sdk/tests/conformance2/query/occlusion-query-scissor.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+<script src="../../js/webgl-test-utils.js"></script>
+</head>
+<body>
+<div id="description"></div>
+<canvas id="canvas" style="width: 50px; height: 50px;"> </canvas>
+<div id="console"></div>
+<script>
+"use strict";
+description("This test verifies the functionality of occlusion query objects with the scissor test.");
+
```
Comment (kenrussell) [inline sdk/tests/conformance2/query/occlusion-query-scissor.html:23]: Could you add a reference to a bug filed somewhere? For example:

`debug('Regression test for <a href="http://anglebug.com/xyzw">http://anglebug.com/xyzw</a>');`

**Review by kenrussell — COMMENTED** [review]
Super work, thanks for distilling this into a reproducible test. Before we merge I'd like to ensure the commit message refers to a bug ID, so can I request a small change in that vein?

**Comment by greggman** [comment]
added bug references

**Review by kenrussell — APPROVED** [review]
Sorry I dropped this on the floor! The bug reference looks good.

Every 16th iteration is failing on M1, on Chrome Canary with ANGLE's Metal backend. Let's investigate why offline.

**Pull request merged**